problems/3.conditionals.js

```diff
@@ -2,25 +2,36 @@
 * @param {number} temp - The current temperature
 * @returns {string} - A message about the temperature
 */
-function temperatureMessage() {
+function temperatureMessage(temp) {
+  if (temp = -32) {
```

Comment:
= is the assignment operator. You want to using comparison operators like temp < 32.